sequencer/src/api.rs

```diff
@@ -1,4 +1,7 @@
-use crate::{network, transaction::Transaction, Header, Leaf, NamespaceProofType, Node, SeqTypes};
+use crate::{
```

Comment:
Ah, ok, I misunderstood something. You have discovered that only the commitment task deals with the HotShot contract, not the sequencers themselves, which is correct. However, the commitment task now runs as a completely separate service. So I don't think we should import commitment task stuff into the sequencer API. Instead, we should add this endpoint to the commitment task API .

Currently, the commitment task currently only has [a stub API](https://github.com/EspressoSystems/espresso-sequencer/blob/main/sequencer/src/bin/commitment-task.rs#L128-L129), which gives us generated healthcheck and version endpoints and nothing else. You could pull that `start_http_server` out into another file and add a non-trivial endpoint like the one you defined here.